_posts/2020-07-24-quadratic-equation-and-programming.markdown

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title:  "Quadratic equation and programming"
+date:   2020-07-24 22:30:41 -0300
+categories: tools
+---
+Programming is the best method ever to solve problems within a machine and whether combined with math, we can do magical things that Harry Potter would be jealous.
+
+I'm not an expert in math, but when I remember some formulas, I search for on google and try to combine with my codes. It helps me create algorithms with better performance - I mean coding faster in this case - and design, so take a look at this example below.
```

Comment:
I could use: "I google it" instead of: "I search for on google".
It's a common expression in english.

Reply:
Nice! Thank you